samples/FunctionApp/Program.cs

```diff
@@ -26,10 +26,7 @@ static async Task Main(string[] args)
                 })
                 //</docsnippet_configure_app>
                 //<docsnippet_middleware>
-                .ConfigureFunctionsWorker((c, b) =>
-                {
-                    b.UseDefaultWorkerMiddleware();
-                })
+                .ConfigureFunctionsWorkerDefaults()
```

Comment:
@ggailey777 this would have documentation impact.

We need to decide if a new sample is required, as this uses the simpler model, but doesn't add middleware.

Reply:
This snippet is currently called out in the [middleware section](https://review.docs.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide?branch=pr-en-us-149017#middleware). I assume this new version isn't as useful in that context.

Reply:
Yeah... I can leave it with middleware registration, but I think a sample with custom middleware would actually better drive the point.